index.js

```diff
@@ -20,15 +36,30 @@ var prefs = Cc["@mozilla.org/preferences-service;1"].getService(Ci.nsIPrefServic
 var Request = require("sdk/request").Request;
 var tabs = require("sdk/tabs");
 
+const DEBUG = true;
+const TAG = "bitcoin-price-ticker";
+const DATA_PROVIDERS_URL = "https://raw.githubusercontent.com/neoranga55/bitcoin-price-ticker/master/data/data-providers.json";
+const ADDON_UPDATE_DOCUMENT_URL = "http://neoranga55.github.io/bitcoin-price-ticker/";
+const ADDON_ID = "jid0-ziK34XHkBWB9ezxd4l9Q1yC7RP0@jetpack";
+const DEFAULT_REFRESH_RATE = 60;
+const DEFAULT_FONT_SIZE = 14;
+const WIDGET_SUFFIX = "-widget";
+const IFRAME_SUFFIX = "-iframe";
```

Comment:
'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).